go.mod

```
@@ -3,25 +3,25 @@ module github.com/gardener/gardener-extension-os-coreos
 go 1.13
 
 require (
-	github.com/gardener/gardener v1.2.1-0.20200408030154-40b97d31d7f7
+	github.com/gardener/gardener v1.6.0
```

Please vendor the latest release.
```suggestion
	github.com/gardener/gardener v1.6.5
```